pkg/apmintegration/helpers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func getGithubOrg(url string) (string, error) {
+	split := strings.Split(url, "/")
```

Comment:
As this is a URL path, also the `path` package seems more appropriate (also has a `Split` func)

Reply:
path is not very good with urls, specially due to the `//` part at the start of it

Reply:
The split function in `path` doesn't do exactly what we need. The `strings` split works better. It might be worth moving entirely to URL tho.

Reply:
The best approach is to use `url.Parse` to get (and validate) a well-formed URL (including eventual password and username, query strings, etc.), then the `path` package can be used to manipulate the `url.Path` part of a URL.

`The path package should only be used for paths separated by forward slashes, such as the paths in URLs` https://pkg.go.dev/path#pkg-overview

Reply:
I switched to using the url package. I use `ParseRequestURI` instead of `Parse` because Parse accepts relative paths and we only want absolute paths.